src/test/java/io/axoniq/axonserver/connector/FakeResultStream.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2022. AxonIQ
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.axoniq.axonserver.connector;
+
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicLong;
+
+/**
+ * Fake implementation of {@link ResultStream} that returns a certain number of {@link Long} elements.
+ * For unit test purpose.
+ *
+ * @author Sara Pellegrini
+ */
+public class FakeResultStream implements ResultStream<Long> {
```

Comment:
I'd recommend using the name  "StubResultStream". "Stub" is commonly used to represent a replacement of a real thing for test purposes.

Reply:
Done